.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@nkorinek when you add files to earthpy, please do not use `git add .` make sure that you are only adding the files that you want to commit to the repo. all of the .pytest_Cache files should not be in this pr. please check this before you ask us to review things!

Reply:
Sorry about these files! My git was being weird about these files, and I had to commit the files in order to update my vignette branch from the earthpy master branch. Should I just delete the file from the pull request?

Reply:
@nkorinek if these are files that occur when you build, we can add them to the .gitignore file. please add this to what we discuss in our meeting this week. if the files are ignored then ou don't have to worry about commiting them by mistake. also please remind me to show you how to "checkout" or remove a changed file from staging in git. `git checkout filename.here`